packages/common/src/utils/regex/RegexUtils.tsx

```diff
@@ -28,4 +28,19 @@ export const RegexUtils = {
     const matcher = new RegExp(substring, 'i');
     return matcher.test(testString);
   },
+  // Case-insensitive version of 'startsWith'
+  matchString: (substring: string, testString: string) => {
```

Comment:
Just wondering if we want to make these two functions (matchString and matchSubstring) have slightly more helpful names. Maybe `matchStringIn` and `matchStringStart`? Just something so it's clear from the function name which is which.